commands.json

```diff
@@ -1898,5 +1898,447 @@
+                "token": "SORTBY"
+              },
+              {
+                "name": "order",
+                "type": "oneof",
+                "optional": true,
+                "arguments": [
+                  {
+                    "name": "asc",
+                    "type": "pure-token",
+                    "token": "ASC"
+                  },
+                  {
+                    "name": "desc",
+                    "type": "pure-token",
+                    "token": "DESC"
+                  }
+                ]
+              }
+            ]
+          },
+          {
+            "name": "nosort",
+            "type": "pure-token",
+            "token": "NOSORT"
+          }
+        ]
+      },
+      {
+        "name": "params",
+        "type": "block",
+        "optional": true,
+        "arguments": [
+          {
+            "name": "params",
+            "type": "pure-token",
+            "token": "PARAMS"
+          },
+          {
+            "name": "nargs",
+            "type": "integer"
+          },
+          {
+            "name": "values",
+            "type": "block",
+            "multiple": true,
+            "arguments": [
+              {
+                "name": "name",
+                "type": "string"
+              },
+              {
+                "name": "value",
+                "type": "string"
+              }
+            ]
+          }
+        ]
+      },
+      {
+        "name": "timeout",
+        "type": "integer",
+        "optional": true,
+        "token": "TIMEOUT"
+      },
+      {
+        "name": "format",
+        "type": "string",
+        "optional": true,
+        "token": "FORMAT"
+      },
+      {
+        "name": "load",
+        "type": "block",
+        "optional": true,
+        "arguments": [
+          {
+            "name": "load",
+            "type": "pure-token",
+            "token": "LOAD"
+          },
+          {
+            "name": "count_or_all",
+            "type": "oneof",
+            "arguments": [
+              {
+                "name": "count",
+                "type": "integer"
+              },
+              {
+                "name": "all",
+                "type": "pure-token",
+                "token": "*"
+              }
+            ]
+          },
+          {
+            "name": "field",
+            "type": "string",
+            "multiple": true,
+            "optional": true
+          }
+        ]
+      },
+      {
+        "name": "groupby",
+        "type": "block",
+        "optional": true,
+        "arguments": [
+          {
+            "name": "groupby",
+            "type": "pure-token",
+            "token": "GROUPBY"
+          },
+          {
+            "name": "nproperties",
+            "type": "integer"
+          },
+          {
+            "name": "property",
+            "type": "string",
+            "multiple": true
+          },
+          {
+            "name": "reduce",
```

Comment:
Shouldn't we have here the same definition of `REDUCE` that we have for `FT.AGGREGATE` ?